### DISCUSSION TITLE
function to simplify adding points3d

### DISCUSSION THREAD

**Discussion by wiesehahn** [r-lidar/lidR#676] [General]
We can plot a pointcloud and add raster or spatial points like:
```R
x <- plot(las)
add_dtm3d(x, dtm)
add_treetops3d(x, ttops)

plot(las) |> add_dtm3d(dtm) |> add_treetops3d(ttops)
```

and we can plot a pointcloud and add another pointcloud like:
```R
x <- plot(las)
plot(las2, add=x)
```

would it make sense to add something like `add_points3d`, so that its possible to apply `plot(las) |> add_points3d(las2)`?
I guess its not necessary and probably not used very often but I was just looking for something like this.

**Comment by Jean-Romain** [comment]
This is definitively doable
